lib/SPIRV/libSPIRV/SPIRVEntry.h

```diff
@@ -560,6 +560,25 @@ class SPIRVMemberName : public SPIRVAnnotation<OpName> {
   std::string Str;
 };
 
+class SPIRVModuleProcessed : public SPIRVAnnotation<OpModuleProcessed> {
+public:
+  static const SPIRVWord FixedWC = 1;
+  // Complete constructor
+  SPIRVModuleProcessed(const SPIRVEntry *TheTarget,
```

Comment:
I'm not sure we should inherit from `SPIRVAnnotation` here. 
What should be given as `TheTarget` argument for `SPIRVModuleProcessed` constructor?

Reply:
Fixed
I used SPIRVEntryNoId as base